packages/sources/finage/test/integration/adapter.test.ts

```diff
@@ -54,6 +54,7 @@ describe('execute', () => {
       data: {
         endpoint: 'uk_etf',
         base: 'IBTA',
+        country: 'uk',
```

Comment:
Can we have a test that checks the `uk_etf` endpoint still returns `uk` data even when no `country` is provided?